docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -71,6 +75,21 @@ logging:
 
 These stats are logged based the number of milliseconds set in your <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which means it could add substantial noise to your logs. Only enable this level of logging temporarily.
```

Comment:
```suggestion
These stats are logged based on the number of milliseconds set in your <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which means it could add substantial noise to your logs. Only enable this level of logging temporarily.
```